src/Microsoft.ML.FastTree/GamTrainer.cs

```diff
@@ -931,6 +931,56 @@ private double GetBinEffect(int featureIndex, double featureValue, out int binIn
             return _binEffects[featureIndex][binIndex];
         }
 
+        /// <summary>
+        /// Get the intercept of the trained model.
+        /// </summary>
+        /// <returns>The intercept of the trained model.</returns>
+        public double GetIntercept() => _intercept;
+
+        /// <summary>
+        /// Get the bin upper bounds for each feature.
+        /// </summary>
+        /// <param name="featureIndex">The index of the feature (in the training vector) to get.</param>
+        /// <returns>The bin upper bounds. May be null if this feature has no bins.</returns>
+        public double[] GetFeatureBinUpperBounds(int featureIndex)
```

Comment:
>GetFeatureBinUpperBounds [](start = 24, length = 24)

As an alternative you can _binUpperBounds an public readonly ImmutableArray<double>[] or ImmutableList<ImmutableList<double>>